ui/src/components/yaml/styles.module.less

```diff
@@ -208,9 +208,7 @@
   .yaml_content_diagnosisPanel {
     box-sizing: border-box;
     width: 400px;
```

Comment:
The max-height property is removed, which may lead to excessive height on larger screens.